Pipfile

```diff
@@ -6,27 +6,11 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-astroid = "==2.3.2"
-certifi = "==2019.9.11"
-chardet = "==3.0.4"
-idna = "==2.8"
-isort = "==4.3.21"
-lazy-object-proxy = "==1.4.2"
-mccabe = "==0.6.1"
-oauthlib = "==3.1.0"
-pylint = "==2.4.3"
 python-dotenv = "==0.10.3"
 requests = "==2.25.0"
```

Comment:
`requests` is already a requirement for `trakt` so not needed here.
Also it's better to avoid future version conflict with trakt requirement. I sugest to remove `requests`.

Reply:
I had impression this project used it directly, but seems only requests_cache is used directly. Will remove.